app/my_script.py

```diff
@@ -1,4 +1,112 @@
 import os
-import pwd
+import sys
+import json
+import logging
+import pandas as pd
 
-print("Hello {} - welcome to my template-world!".format(pwd.getpwuid(os.getuid())[0]))
+# Initialize log
+log = logging.getLogger(__name__)
+
+# Changing working directory so the script can run on relative paths on the files
+os.chdir('../')
+
+# These constants need to be reevaluated if they should be taken out as enviroment variables
+# This will happen when the last of the code is written
+DLR_MRID_FILEPATH = os.getcwd() + '/tests/valid-testdata/DLR_MRID.csv'
+MRID_KEY_NAME = 'LINE_EMSNAME'
+MRID_VALUE_NAME = 'ACLINESEGMENT_MRID'
+MRID_EXPECTED_HEADER_NAMES = ['ACLINESEGMENT_MRID', 'LINE_EMSNAME', 'DLR_ENABLED']
+
+LIMITS_NAME_FILEPATH = os.getcwd() + '/tests/valid-testdata/Limits_other.xlsx'
```

Comment:
```suggestion
LIMITS_NAME_FILEPATH = os.path.dirname(__file__) + '/../tests/valid-testdata/Limits_other.xlsx'
```
Changing directory COULD generate other issues, and will always be relative to where you are in your terminal, so that will only work in specific usecases. Instead just read the path of the current file and move from there.